### DISCUSSION TITLE
"matchGroupName" for extra settings for given group?

### DISCUSSION THREAD

**Discussion by szpak** [renovatebot/renovate#20359] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

34.129.0

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

For a defined `group` I would like to add some extra constrains - e.g. with `matchGroupName`.

### What you tried so far.

I can duplicate the `matchPackagePrefix` (or anything else used in the group definition), but using a group name could be handy.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
Sounds good, please raise a feature request.

**Reply by szpak** [reply]
Reported: https://github.com/renovatebot/renovate/issues/20362
